src/lib/make-toolbox-xml.js

```diff
@@ -636,7 +637,9 @@ const makeToolboxXML = function (categoriesXML) {
     }
 
     everything.push(xmlClose);
-    return everything.join('\n');
+    return everything.join('\n').split('<TARGET_ID>')
+        .join(targetId); // targetIds are designed to not break XML.
+    // @todo consider something less hacky?
```

Comment:
Talked about this offline: instead of the categories being constants, make them functions that take in the target id and use `${...}` string interpolation instead.